### PR TITLE
Feature Client Sending Communication

### PR DESCRIPTION
**Pull request by lukasbischof**

### CODE REVIEW

File: frontend/src/containers/LobbyScreen/LobbyScreen.tsx [hunk 1]
```diff
@@ -22,6 +25,10 @@ class LobbyScreen extends React.Component<LobbyScreenProps> {
   componentDidMount(): void {
     const connection = new Connection('localhost', 4560, (err: Error) => console.log(err));
     connection.initiateHandshake();
+
+    let user = new User();
+    user.setName('Hanspeter');
+    void connection.send(new Message(Protocol.ClientCommands.CONNECTION.CONNECT, user));
```
Comment (lukasbischof) [inline frontend/src/containers/LobbyScreen/LobbyScreen.tsx:31]: For debögging

File: frontend/src/connection/proto/ProtoPayload.ts [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+  ChatMessage,
+  FourInARowTurnPayload,
+  Game,
+  GameEnd,
+  GameList,
+  GameRequest,
+  User
+} from './Types_pb';
+
+export type ProtoPayload = User | GameRequest | Game | GameList | FourInARowTurnPayload | ChatMessage | GameEnd;
```
Comment (TheFehr) [inline frontend/src/connection/proto/ProtoPayload.ts]: Shouldn't FourInARowTurnPayload be just Turn

**Review by TheFehr — CHANGES_REQUESTED** [review]
See my comment in ProtoPayload.ts

**Review by TheFehr — APPROVED** [review]
I like

**Pull request merged**